### ISSUE TITLE
What do you mean by Anchor Autocalibration?

### ISSUE THREAD

**Issue by jonathanrjpereira** [TIERS/dwm1001_localization_and_calibration#2]
Does this mean that you are only creating an inter-anchor Euclidean Distance Matrix?

**Comment by jopequ** [comment]
Hi @jonathanrjpereira and sorry for the late reply. Yes, indeed the firmware only calculates the inter-anchor distances, taking multiple measurements for each pair and returning the average and standard deviation. The actual positions are then calculated on the companion computers, which in turn utilize these positions for tag localization.

**Comment by jonathanrjpereira** [comment]
@jopequ What algorithm is used to find the actual positions of the anchors?

**Comment by jopequ** [comment]
We have only uploaded to GitHub a simple triangulation method in https://github.com/TIERS/uwb-vio-lidar-fusion. We use least squares estimator as described here: https://tiers.utu.fi/paper/martinez2020autocalibration

The repo I linked above does not have proper documentation yet, however. It just contains a set of nodes for localizing a UAV using UWB and extracting the UAV's pointcloud from a lidar scan using PCL. The detailed descriptions are available in here: https://www.utupub.fi/handle/10024/150452
